packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -276,7 +276,8 @@ abstract class FocusTraversalPolicy with Diagnosticable {
   }
 
   // Sort all descendants, taking into account the FocusTraversalGroup
-  // that they are each in, and filtering out non-traversable/focusable nodes.
+  // that they are each in, and filtering out non-traversable/focusable nodes,
+  // but leaving in any group nodes.
```

Comment:
This method-level comment says that group nodes are included in the the returned `List< FocusNode >`, but the implementation comment within the method on line 342ff says they are removed - that's confusing 🤯

Reply:
Good point. Fixed.